### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by xsaurabhsuthar**
- Added Routes.
- Added Routes for Address Management, SingleProductPage, and WishList.
- Added Header component.
- Added ESLint configuration file.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React from "react";
 import { Outlet } from "react-router-dom";
 import "./App.css";
 import { Header } from "./components/Header/Header";
```
Comment (ScoobyDubeyDo) [inline src/App.js]: suggestion:- Try using barrel export

**Review by shobit1337 — APPROVED** [review]
Everything looks clean and good! Good work @Launchpad5682

File: src/App.js (hunk 1 above)
Comment (xsaurabhsuthar) [reply src/App.js]: done

**Review by ScoobyDubeyDo — APPROVED** [review]

**Comment by xsaurabhsuthar** [comment]
Thanks @shobit1337  and @aman-can

**Pull request merged**